Framework/DataHandling/src/CreateChunkingFromInstrument.cpp

```diff
@@ -448,10 +448,9 @@ void CreateChunkingFromInstrument::exec() {
       throw std::runtime_error("Failed to find any banks in the instrument");
 
     // fill in the table workspace
-    for (auto group = grouping.begin(); group != grouping.end(); ++group) {
+    for (auto &group : grouping) {
       stringstream banks;
-      for (auto bank = group->second.begin(); bank != group->second.end();
-           ++bank)
+      for (auto bank = group.second.begin(); bank != group.second.end(); ++bank)
```

Comment:
I think this loop could be converted, too. Was the tool confused by the use of `bank.str()`?